### DISCUSSION TITLE
Taskbar and system tray randomly disappears

### DISCUSSION THREAD

**Discussion by SnakeOilSalesman1880** [valinet/ExplorerPatcher#1440] [Q&A]
A few times a day, my taskbar and system tray will randomly disappear.  The Windows Start menu logo stays visible.  Thankfully, it does come back on its own.  At the same time, it is annoying and I would like to never disappear. 

I am using the latest version of Windows 11 (22H2), and I've installed all of the system updates. 

I am also using the latest version of ExplorerPatcher.

**Comment by pyrates999** [comment]
Please post the build version of your windows that you are running.

**Comment by SnakeOilSalesman1880** [comment]
![image](https://github.com/valinet/ExplorerPatcher/assets/20017086/7d2163e3-4a59-4379-8a75-7ec6b072a873)

**Comment by pyrates999** [comment]
Please check that you've installed this release of explorer patcher: https://github.com/valinet/ExplorerPatcher/releases/tag/22621.1555.55.1_a95a688

A screen shot will do.

**Reply by SnakeOilSalesman1880** [reply]
![image](https://github.com/valinet/ExplorerPatcher/assets/20017086/357cd2d9-fbcc-4dd8-96cc-309d46920b25)

**Comment by pyrates999** [comment]
Before you try any of this, try rebooting your computer.

Then you can try the following:

Uninstall any software you have installed that modifies windows such as 7+ taskbar tweaker and windhawk.  Then reboot.

If it is still happening:

1. make a note of all the settings you've changed
2. uninstall explorer patcher
3. export this registry key to a reg file so you have a backup of it: "Computer\HKEY_CURRENT_USER\Software\ExplorerPatcher"
4. delete the registry key: "Computer\HKEY_CURRENT_USER\Software\ExplorerPatcher"
5. see if it is still happening

If it is not:

1. install explorer patcher again
2. wait for the symbols to download and use the explorer patcher properties to restart windows explorer
3. redo all your settings

If it's now happening again:

1. uninstall explorer patcher
2. delete the registry key: "Computer\HKEY_CURRENT_USER\Software\ExplorerPatcher"
3. download and install the latest pre-release from here: https://github.com/valinet/ExplorerPatcher/releases
4. redo all your settings

If it is still happening: open up an issue here (https://github.com/valinet/ExplorerPatcher/issues) and include the following:

1. windows version
2. explorer patcher version
3. your backup reg file
4. include any other software you have installed that modifies the look of windows like 7+ taskbar tweaker and windhawk
5. if you have windhawk installed, include what mods you have installed

**Reply by SnakeOilSalesman1880** [reply]
Epic thanks!
